sequencer/src/state.rs

```diff
@@ -338,6 +477,12 @@ impl FeeAmount {
     }
 }
 
+impl From<u64> for FeeAmount {
```

Comment:
I think we had this before, but we removed it. Why are we putting it back again?

Reply:
I think it's currently only used here

```
error[E0277]: the trait bound `state::FeeAmount: From<{integer}>` is not satisfied
   --> sequencer/src/api.rs:920:56
    |
920 |         state.prefund_account(Default::default(), 1000.into());
    |                                                        ^^^^ the trait `From<{integer}>` is not implemented for `state::FeeAmount`
    |
    = help: the trait `From<ethers::types::U256>` is implemented for `state::FeeAmount`
    = help: for that trait implementation, expected `ethers::types::U256`, found `{integer}`
    = note: required for `{integer}` to implement `Into<state::FeeAmount>`
    ```

Reply:
I think it's just something that makes sense since `FeeAmount` is supposed to act like a number. E.g. `U256` also implements this

Reply:
Yea I don't think it hurts anything either but you can just do: `U256::from(1000).into()`.